app/services/send_chase_email_to_provider.rb

```diff
@@ -0,0 +1,17 @@
+class SendChaseEmailToProvider
+  def self.call(application_choice:)
+    application_choice.provider.provider_users.each do |provider_user|
+      ProviderMailer.chase_provider_decision(provider_user, application_choice).deliver
+      audit_chase_email(provider_user, application_choice)
+    end
+    ChaserSent.create!(chased: application_choice, chaser_type: :provider_decision_request)
+  end
+
+  def self.audit_chase_email(provider_user, application_choice)
+    course_name_and_code = application_choice.course.name_and_code
+    audit_comment =
+      "Chase emails have been sent to the provider #{provider_user.email_address} because " +
+      "the application for #{course_name_and_code} is half way through its RBD period."
```

Comment:
With the `TimeLimitConfig` setup we have it's possible that we could tweak things so that it's not half way.

Reply:
What do you think about `...is close to its RBD date.`?

Reply:
could we say `is less than #{TimeLimitConfig...} days from it's RBD date`? (I'm not a content person!)